go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/StackExchange/dnscontrol/v4
 
-go 1.25
+go 1.25.3
```

Comment:
Was that change intentional?

The latest version is 1.25.4 as of Nov 5: https://go.dev/doc/devel/release#go1.25.4